fileProcessing.py

```diff
@@ -105,15 +106,16 @@ def thermalRaw_toMp4():
         'newProcessedFileKey': newKey,
         'result': json.dumps(result),
     }
+    success = False
     try:
         r = requests.put(API_URL, data = params)
     except:
         print("Error with connecting to server.")
-        return False
     if r.status_code == 200:
         print("Finished file processing")
-        return True
-    return False
+        success = True
+    shutil.rmtree(folder)
```

Comment:
It would be better to use a try/finally to ensure the folder is cleaned up regardless of exceptions or early returns.